docs/ReduxIntegration.md

```diff
@@ -0,0 +1,12 @@
+---
+id: redux-integration
+title: Redux Integration
+---
+
+Codesandbox provided as proof.
+
+<iframe
+  src="https://codesandbox.io/embed/new?codemirror=1"
```

Comment:
I requested creating an account, will update this PR once done

Reply:
We have an account now. So a step forward to this, would be for you to create a codesandbox with Redux example, which I could then fork to the official account